### DISCUSSION TITLE
Fake Screen Resolution

### DISCUSSION THREAD

**Discussion by il1ze** [seleniumbase/SeleniumBase#3101] [General]
How can I fake Screen Resolution in local selenium? Maybe you have already encountered this problem and know the solution?

**Comment by mdmintz** [comment]
You can use `set_window_size(width, height)` to *change* your screen *size* (if that's what you mean):

https://github.com/seleniumbase/SeleniumBase/blob/4c455c46752e36ab6d469f6f293e09257178206a/help_docs/method_summary.md?plain=1#L238

Also see the [Mobile Mode docs](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/mobile_testing.md) if you want to set settings for mobile testing.

**Comment by il1ze** [comment]
Thanks for the answer, but by screen resolution I mean screen (display) size
![image](https://github.com/user-attachments/assets/f95fee16-5537-4205-a6a6-36c9c6aef25a)

**Reply by mdmintz** [reply]
You can’t fake the resolution. You can change the display width and height with the built-in methods. The mobile emulator lets you set some display settings such as width, height, and pixel ratio.

**Reply by il1ze** [reply]
Thank you
